<launch>

	<arg name="config_file" default="$(find recognize_core_libs)/../ThirdParty/darknet/cfg/yolov3.cfg"/>
	<arg name="weight_file" default="$(find recognize_core_libs)/../ThirdParty/darknet/cfg/yolov3.weights"/>
	<arg name="names_file" default="$(find recognize_core_libs)/../ThirdParty/darknet/data/yolov3.labels"/>

	<node pkg="recognize_detection" type="detection_node" name="detection" output="screen">
    <param name="yolo_config_file" value="$(arg config_file)" />
    <param name="yolo_weight_file" value="$(arg weight_file)" />
    <param name="yolo_labels_file" value="$(arg names_file)" />
    <param name="capture_mode" value="ROS" />
    <param name="rate" type="double" value="1.0" />
    <param name="detection_threshold" type="double" value="0.8" />

    <remap from="~input" to="/camera/image_raw"/>
  </node>
</launch>
